spec/active_yaml/base_spec.rb

```diff
@@ -20,13 +20,25 @@ class Empty        < ActiveYaml::Base ; end # Empty YAML
     Object.send :remove_const, :ArrayRow
     Object.send :remove_const, :City
     Object.send :remove_const, :State
+    Object.send :remove_const, :User
     Object.send :remove_const, :Empty
   end
 
   describe ".load_path" do
-    it 'can execute embedded ruby' do
-       User.first.email.should =~ /^user[0-9]*@email.com$/
-       User.first.password.should == 'secret'
+    context 'default' do
+      it 'can execute embedded ruby' do
+        User.first.email.should =~ /^user[0-9]*@email.com$/
+        User.first.password.should == 'secret'
+      end
+    end
+
+    context 'erb disabled' do
+      before { ActiveYaml::Base.set_erb_disabled true }
```

Comment:
Want to set it back otherwise we'll have sporadic test failures

```suggestion
      before { ActiveYaml::Base.process_erb = false }
      after  { ActiveYaml::Base.process_erb = true }
```